crds/AzureBackendPool.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azurebackendpools.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
+  versions:
+    - name: v1
```

Comment:
v1 -> v1alpha1 ?

Reply:
> v1 -> v1alpha1 ?
Sounds good.
Here is some link I find regarding the versioning
https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/

Reply:
will check this out